infra/recipes/terraform/azure/README.md

```diff
@@ -0,0 +1,5 @@
+## Instructions to configure and run the Azure Sandbox
+
+Install the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/), and ensure that you are logged in with your Azure account by running `az login`.
```

Comment:
I was referring to https://github.com/Azure/terraform-azurerm-vnet/tree/main and https://docs.microsoft.com/en-us/azure/virtual-machines/linux/terraform-install-configure. Do we need to add Azure credentials such as subscriptions, tenant, client id, secrets etc. If so can we add in the wiki?

Reply:
I don't think so. I think logging in to your azure account with the Azure CLI covers this.

Reply:
Got it. Azure Cli has different authentication mechanisms. Can you mention in the README  which method is used as per the wiki - https://learn.microsoft.com/en-us/cli/azure/reference-index?view=azure-cli-latest#az-login

Reply:
Also it would be good to have azure account setup instructions in the README file.

Reply:
Added to the README.